php/class-fm-overlays.php

```diff
@@ -181,18 +180,44 @@ public function destroy_transient( $post_id ) {
 	 * Get the conditional values for an overlay
 	 *
 	 * @param int $overlay_id
-	 *
 	 * @return bool|mixed
 	 */
 	public function get_conditionals( $overlay_id ) {
 		$conditionals = get_post_meta( $overlay_id, 'fm_overlays_conditionals', true );
 		return ( empty( $conditionals ) ) ? false : $conditionals;
 	}
 
+
+	/**
+	 * Sets cookie flagging overlay as viewed by client
+	 *
+	 * @param int $overlay_id
+	 */
+	public function set_overlay_cookie( $overlay_id ) {
+		$cookie_name = $this->get_overlay_cookie_name( $overlay_id );
+		/**
+		 * Cookie will expire in 20 Hours
+		 */
+		setcookie( $cookie_name, true, time() + 60 * 60 * 20 );
```

Comment:
Yeah, I will add something in the README about this. Ultimately, this should fall to the individual site to address, not the plugin. The plugin _should_ reiterate this in the README.